OneTimePassword/Token.swift

```diff
@@ -8,6 +8,8 @@
 
 import Foundation
 
+let Z = "Z"
```

Comment:
Name Format Violation: Variable name should start with a lowercase character: 'Z'